buggy.go

```diff
@@ -0,0 +1,71 @@
+/*
+ * Buggy, (C) 2016,2017 Minio, Inc.
```

Comment:
This should be just 2017, since its a new file - perhaps call this `main.go` as filename.